twoliter/embedded/Makefile.toml

```diff
@@ -354,11 +354,29 @@ dependencies = ["setup"]
 script_runner = "bash"
 script = [
 '''
+declare -a CARGO_MANIFESTS
+declare -a CARGO_DIGESTS
+mkdir -p .cargo/vendor
+if [ ! -f ".cargo/config.toml" ]; then
+    touch .cargo/config.toml
+fi
 for ws in sources variants .; do
   [ -s "${BUILDSYS_ROOT_DIR}/${ws}/Cargo.toml" ] || continue
-  cargo fetch --locked --manifest-path "${BUILDSYS_ROOT_DIR}/${ws}/Cargo.toml"
+  CARGO_MANIFESTS+=(--sync "${BUILDSYS_ROOT_DIR}/${ws}/Cargo.toml")
+  CARGO_DIGESTS+=("$(sha256sum "${BUILDSYS_ROOT_DIR}/${ws}/Cargo.lock" | awk '{ print $1 }')")
```

Comment:
I wonder what happens if `Cargo.lock` doesn't exist? I think this is possible if a user does not have different workspaces for each of these directories.

As we discussed we can just use Cargo.lock as the marker file to make it work.